creation/server/creation.lua

```diff
@@ -35,42 +35,81 @@ function printoutHeader(name)
 end
 
 function parsePoly(zone)
-  local printout = printoutHeader(zone.name)
-  printout = printout .. "PolyZone:Create({\n"
-  for i=1, #zone.points do
-    if i ~= #zone.points then
-      printout = printout .. "  vector2(" .. tostring(zone.points[i].x) .. ", " .. tostring(zone.points[i].y) .."),\n"
-    else
-      printout = printout .. "  vector2(" .. tostring(zone.points[i].x) .. ", " .. tostring(zone.points[i].y) ..")\n"
+  if Config.ConfigFormat then
```

Comment:
This needs to be updated to `ConfigFormatEnabled` as well (in parseCircle and parseBox too).

Reply:
Whoops, moved to fast for my own good. Should be good now